### PR TITLE
[BW - 50] Feature/adding ui tests

### PR DESCRIPTION
**Pull request by FesenkoG**

### CODE REVIEW

File: BankexWallet/InterfaceElements/DesignElements/Settings/SettingsController.swift [hunk 1]
```diff
@@ -59,6 +59,9 @@ MFMailComposeViewControllerDelegate {
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: settingsToShow[indexPath.row], for: indexPath)
+        if settingsToShow[indexPath.row] == "WalletsListCell" {
+            cell.accessibilityIdentifier = "WalletsListCell"
```
Comment (K0rka) [inline BankexWallet/InterfaceElements/DesignElements/Settings/SettingsController.swift]: Can you set it directly in cell?

File: BankexWalletUITests/SendTokenTests.swift [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        continueAfterFailure = false
+        
+    }
+    
+    override func tearDown() {
+        Springboard.deleteMyApp()
+        super.tearDown()
+    }
+    
+    func testExample() {
```
Comment (K0rka) [inline BankexWalletUITests/SendTokenTests.swift]: Can you please, change the name

File: BankexWallet/InterfaceElements/DesignElements/Settings/SettingsController.swift (hunk 1 above)
Comment (FesenkoG) [reply BankexWallet/InterfaceElements/DesignElements/Settings/SettingsController.swift]: Yes, fixed it.

File: BankexWalletUITests/SendTokenTests.swift (hunk 2 above)
Comment (FesenkoG) [reply BankexWalletUITests/SendTokenTests.swift]: Yes, fixed.

**Comment by K0rka** [comment]
@FesenkoG Resolve conflicts, please, to merge this

**Pull request merged**